release

```diff
@@ -580,7 +580,7 @@ cp "$TMP_DIR/$BASENAME/PackageInfo.g" .
 rm -rf doc/ htm/
 if [ -f "$TMP_DIR/$BASENAME/doc/chap0.html" ] ; then
     mkdir -p doc/
-    for ext in css html js txt ; do
+    for ext in css html js png txt ; do
```

Comment:
This will trigger an error if there is no png file

Reply:
Ah, indeed.

Reply:
Perhaps the -f option helps, but One would need to check it is portable. Alternatively, an `if` checking for any files first.

On the long run, even better would be a standardized way for packages to "install" their docs